staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1092,8 +1092,10 @@ type SecretVolumeSource struct {
 	// relative and may not contain the '..' path or start with '..'.
 	// +optional
 	Items []KeyToPath `json:"items,omitempty" protobuf:"bytes,2,rep,name=items"`
-	// Optional: mode bits to use on created files by default. Must be a
-	// value between 0 and 0777. Defaults to 0644.
+	// Optional: mode bits used to set permissions on created files by default.
+	// Must be an octal value between 0000 and 0777 or a decimal value between 0 and 511.
+	// Yaml accepts both octal and decimal values, json requires decimal values
```

Comment:
this is way more helpful, thanks! one nit, use `YAML` and `JSON`, then lgtm (applies everywhere)